sdks/python/tox.ini

```diff
@@ -88,6 +89,21 @@ commands =
   pip --version
   time {toxinidir}/run_pylint.sh
 
+[testenv:py27-lint3]
```

Comment:
Can we add a comment how this is different from py3-lint? Or perhaps we don't need both of them?

Reply:
py27-lint3 checks for portability issues, while py3-lint checks for python 3 issues. I'll add a comment to clarify.